nvm.sh

```diff
@@ -3573,11 +3584,16 @@ nvm() {
         >&2 nvm --help
         return 127
       fi
-
+      local NVMRC_SAVE_TEXT
+      NVMRC_SAVE_TEXT=''
+      if [ "${NVM_WRITE_TO_NVMRC}" -eq 1 ] && [ "${VERSION}" != '∞' ] && [ "${VERSION}" != 'N/A' ]; then
+        echo "$VERSION" | tee "${PWD}"/.nvmrc > /dev/null || nvm_err "WARNING: Unable write version to .nvmrc"
```

Comment:
Should this line `exit 3` on error or is a warning sufficient?

Reply:
Yes, i think it should.